challenge_3/cpp/blakebarnes00/src/main.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+void print_array(std::vector<int> array)
```

Comment:
It's a way to pass parameters by pointer without all of the special pointer syntax. Since print_array is your only method, that's the only location it's applicable.